pages/blueprint/index.js

```diff
@@ -393,27 +393,11 @@ class BlueprintPage extends React.Component {
             <div className="tab-container row">
               <div className="form-customizations col-sm-12 pf-u-ml-md">
                 <div className="form-horizontal">
-                  <div
-                    id="input-hostname"
-                    className={`form-group ${editHostnameInvalid ? "has-error" : ""}`}
-                    data-form="hostname"
-                    onSubmit={() => this.handleEditHostname("commit")}
-                  >
+                  <div className="form-group" id="input-hostname">
```

Comment:
Why is this `id="input-hostname"` and not simply `id="hostname"`?

Reply:
this id is just a leftover from when these divs where text inputs.